### ISSUE TITLE
Need to save detailed outputs from S5

### ISSUE THREAD

**Issue by taylorbell57** [kevin218/Eureka#117]
Before we start a Stage 6 (Issue #98), we need to save all the detailed outputs from S5 which we currently don't do. One issue is that Parameter objects aren't pickleable (as will be many other things), so we'll need to work out how to cleanly save the outputs

**Comment by taylorbell57** [comment]
We could save Parameters objects (and LightCurve objects) with a bit of tweaking, but we cannot do so if we are reloading the classes as reloaded classes are not pickleable

**Comment by kevin218** [comment]
Can you help me understand why these objects can't be pickled. I've never had trouble in the past using `manageevent.py`.

**Comment by taylorbell57** [comment]
My initial attempts directly using pickle gave errors saying something to the effect of `<eureka.S5_lightcurve_fitting.parameters.Parameters> is not the same as <eureka.S5_lightcurve_fitting.parameters.Parameters>` which only went away when I removed the reload statements. I didn't try using the manageevent.py method yet though. I think the solution we merged in with PR #93 was a cleaner solution anyways though - I output the median values (or best-fit values from lsq) into a csv file as well as the chains from dynesty or emcee into a csv. That way things are broadly accessible to anyone, human and machine readable, and should be all I need to write the Stage 6 code. I was going to close this issue with that PR, but wanted to respond before deciding whether we should close this or dig further into trying to save the LightCurve object
